cnd/src/swap_protocols/dependencies.rs

```diff
@@ -0,0 +1,15 @@
+use btsieve::{bitcoin::BitcoindConnector, ethereum::Web3Connector};
```

Comment:
:mega: This file is not actually declared in the `mod.rs` of this module, so it's being ignored.

Reply:
You have really good attention to detail, thanks - now removed.